From 6d0ea02a4942a8454cf733669f3ecea9899532cb Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Wed, 26 Jul 2017 10:58:14 -0700
Subject: [PATCH] VOLAPI-62 Remove all references to state `rolling_back` in
 current implementation Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
 Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 lib/validation/volumes.js | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/lib/validation/volumes.js b/lib/validation/volumes.js
index a77c3c3..1e14d2b 100644
--- a/lib/validation/volumes.js
+++ b/lib/validation/volumes.js
@@ -148,9 +148,7 @@ function validateVolumeSizeSearchParam(size) {
 function validateVolumeState(state) {
     console.log('state: ', state);
 
-    var VALID_STATES = [
-        'creating', 'ready', 'failed', 'rolling_back', 'deleting'
-    ];
+    var VALID_STATES = ['creating', 'ready', 'failed', 'deleting'];
     var err;
 
     if (VALID_STATES.indexOf(state) === -1) {
-- 
2.21.0

